src/core/client_channel/retry_throttle.cc

```diff
@@ -44,26 +60,24 @@ ServerRetryThrottleData::ServerRetryThrottleData(
   // we will start out doing the same thing on the new one.
   if (old_throttle_data != nullptr) {
     double token_fraction =
-        static_cast<uintptr_t>(
-            gpr_atm_acq_load(&old_throttle_data->milli_tokens_)) /
+        static_cast<double>(
+            old_throttle_data->milli_tokens_.load(std::memory_order_relaxed)) /
```

Comment:
This load is changing from acquire to relaxed, and the store below is changing from release to relaxed.  Are these changes safe?

Reply:
Yeah, we're not ordering memory on accesses to `milli_tokens_`